lib/gyro/json/generator.rb

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 - Niji
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'gyro/xcdatamodel/parser'
+require 'json'
+
+module Gyro
+  module Json
+    # Generates the JSON representation of the input datamodel
+    #
+    class Generator
+      # INCLUDES #############################################################
+
+      include Gyro::XCDataModel::Parser
```

Comment:
I don't think this is actually necessary.

I know that in the old code we had that `include Gyro::XCDataModel::Parser` too already, but I think it was bad ruby practice and structure, and in practice doesn't make any sense and can actually be completely removed I think.